src/Aquarius.Client/TimeSeries/Client/generate_code_from_live_endpoint.sh

```diff
@@ -39,6 +39,12 @@ echo "Generating $OutputFile ..."
 OutputFile=$OutputPath/$EndPointName.cs
 curl -s -o "$OutputFile" "http://$ServerName/AQUARIUS/$EndPoint/types/csharp?MakePartial=false&MakeVirtual=false&ExportValueTypes=true&GlobalNamespace=$GlobalNamespace&DefaultNamespaces=System,System.Collections.Generic,ServiceStack,ServiceStack.DataAnnotations,ServiceStack.Web,NodaTime" || exit_abort "Can't read endpoint"
 
+# 2021.1 added a hidden IFileUploadRequest.IsFileRequired property with [ApiMember(ExcludeInSchema = true)][IgnoreDataMember] attributes.
```

Comment:
@MartinBenn-AI This is the hack, since it looks like the ServiceStack code generator is not respecting either the `[ApiMember(ExcludeInSchema = true)]` or the `[IgnoreDataMember]` attributes. I'm not sure which.

So the hack is to just comment out the `IsFileRequired` property from the generated `IFileUploadRequest` interface.

The small risk with this hack is that it comments out *any* boolean `IsFileRequired` property, irrespective of the owning class or interface. So if another interface ever needs to add a property with that same type and name, then this hack will need to be revisited.

Reply:
Yeah, I don't think there is anything you should change. At some point, the SDK should probably switch over to generating service models from the Open API spec, and then this problem would go away (and be replaced by plenty of others, yikes!)